src/test/java/org/broadinstitute/hellbender/tools/spark/BaseRecalibratorSparkShardedIntegrationTest.java

```diff
@@ -70,15 +70,12 @@ private String getCloudInputs() {
         final String HiSeqBam_chr20 = localResources + WGS_B37_CH20_1M_1M1K_BAM;
         final String dbSNPb37_chr20 = localResources + DBSNP_138_B37_CH20_1M_1M1K_VCF;
         final String GRCh37RefLocal = b37_reference_20_21;
+        final String moreSites = getResourceDir() + "bqsr.fakeSitesForTesting.b37.chr17.vcf"; //for testing 2 input files (FIXME - this file is bogus because uses chr17)
 
         return new Object[][]{
                 // local computation and files (except for the reference)
                 {new BQSRTest(GRCh37Ref, HiSeqBam_chr20, dbSNPb37_chr20, "", localResources + BQSRTestData.EXPECTED_WGS_B37_CH20_1M_1M1K_RECAL)},
-
-                // Currently disabled because BaseRecalibratorSpark can't handle more than 1 knownSites file: https://github.com/broadinstitute/gatk/issues/1085
-                // Re-enable once this is fixed.
-                //{new BQSRTest(GRCh37Ref, HiSeqBam_chr20, dbSNPb37, "-knownSites " + moreSites, localResources + "expected.CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.2inputs.recal.txt")},
-
+                {new BQSRTest(GRCh37Ref, HiSeqBam_chr20, dbSNPb37_chr20, "-knownSites " + moreSites, localResources + "expected.CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.2inputs.recal.txt")},
```

Comment:
I've replaced the moreSites file used here with one on chr20 called more20Sites, and replaced the results file with results obtained by running the new input through the walker version of BQSR.